src/main/java/com/onaio/steps/utils/CrashlyticsTree.java

```diff
@@ -0,0 +1,35 @@
+package com.onaio.steps.utils;
+
+import android.util.Log;
+
+import androidx.annotation.Nullable;
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics;
+
+import org.jetbrains.annotations.NotNull;
+
+import timber.log.Timber;
+
+public class CrashlyticsTree extends Timber.DebugTree {
+
+    private static final String CRASHLYTICS_KEY_PRIORITY = "priority";
+    private static final String CRASHLYTICS_KEY_TAG = "tag";
+    private static final String CRASHLYTICS_KEY_MESSAGE = "message";
+
+    @Override
+    protected void log(int priority, @Nullable String tag, @NotNull String message, @Nullable Throwable t) {
```

Comment:
change the variable name `t` to `throwable`

Reply:
> change the variable name `t` to `throwable`

done